benchmark/coniql_performance_test.py

```diff
@@ -0,0 +1,341 @@
+import argparse
+import asyncio
+import datetime
+import json
+import sys
+import threading
+import time
+
+import psutil
+import websockets
```

Comment:
I have included these dependencies in the `pyproject.toml` and can then use the same venv to run the performance tests instead of having to create a new one and tidy up.